kp_x_: 0.8
kp_y_: 0.5
kp_z_: 1.9

ki_x_: 0.005
ki_y_: 0.005
ki_z_: 0.002

kd_x_: 0.3
kd_y_: 0.3
kd_z_: 0.5

pose_x: 10.0
pose_y: 5.0
pose_z: 3.0